pinc/ProjectState.inc

```diff
@@ -325,7 +303,11 @@ function sql_collator_for_project_state($state_column)
 }
 
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
-$project_status_descriptors = ['posted', 'PPd', 'proofed', 'created'];
+
+function get_project_status_descriptors()
+{
+    return ['posted', 'PPd', 'proofed', 'created'];
```

Comment:
Just making sure this doesn't need to include `'deleted'`.

Reply:
None of the current consumers expect it to, so no.